packages/perennial/hardhat.config.ts

```diff
@@ -1,14 +1,14 @@
 import { dirname } from 'path'
 
-import defaultConfig, { OPTIMIZER_ENABLED, SOLIDITY_VERSION } from '../common/hardhat.default.config'
+import defaultConfig, { OPTIMIZER_ENABLED } from '../common/hardhat.default.config'
 const eqPerennialOracleDir = dirname(require.resolve('@equilibria/perennial-oracle/package.json'))
 
 // This Solidity config produces small contract sizes, and is useful when
 // contracts are close to the maximum possible size. The trade off is each
 // function call will likely use extra gas.
 // Mostly inspired by Compound Comet's setup: https://github.com/compound-finance/comet/blob/main/hardhat.config.ts#L124
 const MINIMUM_CONTRACT_SIZE_SOLIDITY_OVERRIDES = {
-  version: SOLIDITY_VERSION,
+  version: '0.8.17',
```

Comment:
was it intentional to hardcode this now?

Reply:
This is just for the min size contracts. Applies to your comment below as to why we changed it to `^0.8.15`, but I didn't want to upgrade everything here to 0.8.17 since we aren't redeploying any other contracts so the default still remains `0.8.15`

Reply:
Got it makes sense 💪